modules/placement-driver/src/main/java/org/apache/ignite/internal/placementdriver/leases/LeaseTracker.java

```diff
@@ -138,10 +175,16 @@ public CompletableFuture<Void> onUpdate(WatchEvent event) {
 
                 if (msEntry.empty()) {
                     leases.remove(grpId);
+                    tryRemoveTracker(grpId);
                 } else {
                     Lease lease = fromBytes(msEntry.value());
 
+                    assert lease.getExpirationTime() != MAX_VALUE : "INFINITE lease expiration time isn't expected";
```

Comment:
The same as above, using of MAX_VALUE is unexpected.

Reply:
Also removed.